package.json

```diff
@@ -8,6 +8,16 @@
     "type": "git",
     "url": "https://github.com/juju/juju-gui.git"
   },
+  "scripts": {
+    "build": "webpack --config webpack.dev.js && cp -r node_modules/yui static && rm -rf static/yui/node_modules && ./scripts/version.sh",
+    "clean": "rm -rf static/yui static/build node_modules webapp/__pycache__",
+    "watch": "webpack --config webpack.dev.js --watch",
+    "lint-js": "eslint --rulesdir eslint-rules jujugui/static/gui/src",
+    "lint": "npm run lint-js",
+    "serve": "./entrypoint 0.0.0.0:$PORT",
+    "test-old": "./scripts/test-js.sh && ./scripts/test-js-old.sh",
```

Comment:
Why do we need this?

Reply:
It's a long story, but we effectively need these old test runners for now. I'll be happy to explain it to you offline.